noop/unikernel.ml

```diff
@@ -0,0 +1,6 @@
+module Main = struct
```

Comment:
If `mirage-skeleton` is to serve the purpose of "tutorial" examples, then there should be a procession of unikernels. I could introduce these under `noop` as is done for `lwt`, but then whither `hello`? The ones I think are obvious would be:

* a true `noop` without functors
* a `noop` using `Log`
* a `noop` with functors and `Log` [ this actually being what `hello` should be ]

Perhaps the last two should replace the current `hello` instead?